crates/ide/src/lib.rs

```diff
@@ -535,6 +535,14 @@ impl Analysis {
         self.with_db(|db| references::rename::prepare_rename(db, position))
     }
 
+    pub fn will_rename_file(
+        &self,
+        file_id: FileId,
+        new_name: &str,
```

Comment:
```suggestion
        new_name_stem: &str,
```

or some equivalent name to make it clear that `.rs` is not included.